skyportal/handlers/api/observation.py

```diff
@@ -443,6 +455,78 @@ def get_observations(
                 intarea = 0.0
 
     total_matches = session.scalar(sa.select(sa.func.count()).select_from(obs_query))
+
+    order_by = None
+    if sort_by is not None:
+        if sort_by == "obstime":
+            order_by = (
+                [Observation.obstime]
+                if sort_order == "asc"
+                else [Observation.obstime.desc()]
+            )
+        elif sort_by == "observation_id":
+            order_by = (
+                [Observation.observation_id]
+                if sort_order == "asc"
+                else [Observation.observation_id.desc()]
+            )
+        elif sort_by == "exposure_time":
+            order_by = (
+                [Observation.exposure_time]
+                if sort_order == "asc"
+                else [Observation.exposure_time.desc()]
+            )
+        elif sort_by == "seeing":
+            order_by = (
+                [Observation.seeing]
+                if sort_order == "asc"
+                else [Observation.seeing.desc()]
+            )
+        elif sort_by == "airmass":
+            order_by = (
+                [Observation.airmass]
+                if sort_order == "asc"
+                else [Observation.airmass.desc()]
+            )
+        elif sort_by == "limmag":
+            order_by = (
+                [Observation.limmag]
+                if sort_order == "asc"
+                else [Observation.limmag.desc()]
+            )
+        elif sort_by == "filt":
+            order_by = (
+                [Observation.filt] if sort_order == "asc" else [Observation.filt.desc()]
+            )
+        elif sort_by == "instrument_name":
+            order_by = (
+                [Observation.instrument_id]
+                if sort_order == "asc"
+                else [Observation.instrument_id.desc()]
+            )
+        elif sort_by == "queue_name":
+            order_by = (
+                [Observation.queue_name]
+                if sort_order == "asc"
+                else [Observation.queue_name.desc()]
+            )
+        elif sort_by == "validity_window_start":
+            order_by = (
+                [Observation.validity_window_start]
+                if sort_order == "asc"
+                else [Observation.validity_window_start.desc()]
+            )
+        elif sort_by == "validity_window_end":
+            order_by = (
+                [Observation.validity_window_end]
+                if sort_order == "asc"
+                else [Observation.validity_window_end.desc()]
+            )
+
+    if order_by is None:
+        order_by = [Observation.instrument_id.desc()]
+    obs_query = obs_query.order_by(*order_by)
```

Comment:
Would this work without the `*` if you also don't put the `order_by` in a list?

Reply:
Yes nice thinking.